### PR TITLE
remove simplification of monomial constants in posynomial

### LINKED ISSUE #1060: incorrect sensitivity mapping on trivial example
Urgent bug.

Here's a simple MWE:

```python
import numpy as np
import pandas as pd
from gpkit import Model, Variable, units


class Aircraft(Model):
    def setup(self):
        W_payload = Variable("W_{payload}", 175*(195 + 30), "lbf", "payload weight")
        f_oew = Variable("f_{oew}", 0.53, "-", "OEW/MTOW")
        fuel_per_nm = Variable("\\theta_{fuel}", 13.75, "lbf/nautical_mile", "fuel burn per nm")
        R = Variable("R", 3000, "nautical_miles", "range")
        mtow = Variable("MTOW", "lbf", "max take off weight")
        self.cost = 61.3e6*units.USD*(mtow/(1e5*units.lbf))**0.807
        return (mtow >= W_payload + f_oew*mtow + fuel_per_nm*R),
    
m = Aircraft()
sol = m.solve()
print sol.table()
print "sens to Wpay: %s" % sol["sensitivities"]["constants"]["W_{payload}"]
```

The result is (note sensitivity to W_payload should be non-zero):

```python
Using solver 'mosek'
Solving for 1 variables.
Solving took 0.0116 seconds.

Cost
----
 9.475e+07 [USD] 

Free Variables
--------------
MTOW : 1.715e+05  [lbf] max take off weight

Constants
---------
            R : 3000       [nmi]     range
  W_{payload} : 3.938e+04  [lbf]     payload weight
theta_{fuel} : 13.75      [lbf/nmi] fuel burn per nm
      f_{oew} : 0.53                 OEW/MTOW

Sensitivities
-------------
            R : +0.41 range
theta_{fuel} : +0.41 fuel burn per nm
      f_{oew} : +0.39 OEW/MTOW
  W_{payload} : +0    payload weight

sens to Wpay: 0.0
```

### PR DESCRIPTION
**Pull request by bqpd**
Closes #1060

### CODE REVIEW

**Comment by bqpd** [comment]
test models please

**Comment by bqpd** [comment]
Looks like one of the engine tests doesn't run with cvxopt...Anyway, @whoburg, this is ready for review and merging.

File: gpkit/nomials/nomial_math.py [hunk 1]
```diff
@@ -494,7 +491,7 @@ def _gen_unsubbed(self):
                              "be converted to units of %s" %
                              (self.p_lt, self.m_gt))
 
-        p.exps, p.cs = self._simplify_posy_ineq(p.exps, p.cs)
```
Comment (whoburg) [inline gpkit/nomials/nomial_math.py]: Not clear to me why we should remove simplification. Seems like this was a mapping bug, not a simplification bug, right? Isn't this only an indirect fix?

**Comment by bqpd** [comment]
Yup, I have a working branch which addresses this better. Will push soon!

File: gpkit/nomials/nomial_math.py (hunk 1 above)
Comment (bqpd) [inline gpkit/nomials/nomial_math.py]: It was both!

**Comment by bqpd** [comment]
Superseded by #1064

**Pull request closed without merging**